enzy_htp/structure/ligand_io/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""The I/O submodule for generation/construction of Ligand objects from files and other data structures. The Ligand() class uniquely needs 
+I/O methods as bond information is frequently needed for these molecules given their customized topologies.
+
+Author: Chris Jurich <chris.jurich@vanderbilt.edu>
+Date: 2023-10-05
+"""
```

Comment:
We should also have mol2_io.py under /structure_io
The submodule stands for "structure" in general including ligand.